kroxylicious-integration-tests/src/test/java/io/kroxylicious/proxy/TlsIT.java

```diff
@@ -352,6 +355,234 @@ void downstreamAndUpstreamTls(Class<? extends PasswordProvider> providerClazz, @
         }
     }
 
+    @Test
+    void downstreamMutualTls_SuccessfulTlsWithProtocolsAllowed(KafkaCluster cluster) throws Exception {
```

Comment:
You've named the test "mutual", but I don't think we actually need mutual authentication in this test.   I'd get rid of the word "mutual" from the new tests, and remove the  client's `SSL_KEYSTORE` configuration.